components/shared/publizonReader/PublizonReader.tsx

```diff
@@ -38,26 +38,25 @@ const Reader = ({ type, onBackCallback, identifier, orderId }: ReaderType) => {
     }
   }, [onBackCallback])
 
-  if (type === "rent") {
+  if (type === "loan") {
     return (
-      <div>
-        <p>orderId: {orderId}</p>
-        <div
-          id="pubhub-reader"
-          order-id={orderId}
-          role="button"
-          tabIndex={0}
-          // eslint-disable-next-line no-script-url
-          close-href="javascript:window.onReaderBackCallback()"
-          aria-label="Go back"
-        />
-      </div>
+      <div
+        style={{ height: "100vh" }}
+        id="pubhub-reader"
+        order-id={orderId}
+        role="button"
+        tabIndex={0}
+        // eslint-disable-next-line no-script-url
```

Comment:
Usually we write a comment when we bypass our own rules. I think we should do that here as well.